airflow/providers/google/cloud/hooks/looker.py

```diff
@@ -0,0 +1,308 @@
+
+        # TODO: Temporary API response stub. Use Looker API once looker sdk 22.2 released.
+        resp = MaterializePDT(materialization_id='366', resp_text=None)
+        # unpack query_params dict into kwargs (if not None)
+        # if query_params:
+        #     resp = sdk.start_pdt_build(model_name=model, view_name=view, source=self.source, **query_params)
+        # else:
+        #     resp = sdk.start_pdt_build(model_name=model, view_name=view, source=self.source)
+
+        self.log.info("Start PDT build response: '%s'.", resp)
+
+        return resp
+
+    def check_pdt_build(
+        self,
+        materialization_id: str,
+    ) -> MaterializePDT:
+        """
+        Gets the PDT materialization job status from Looker.
+
+        :param materialization_id: Required. The materialization id to check status for.
+        """
+        self.log.info("Requesting PDT materialization job status. Job id: %s.", materialization_id)
+
+        # TODO: Temporary API response stub. Use Looker API once looker sdk 22.2 released.
+        resp = MaterializePDT(
+            materialization_id='366',
+            resp_text='{"status":"running","runtime":null,'
+            '"message":"Materialize 100m_sum_aggregate_sdt, ",'
+            '"task_slug":"f522424e00f0039a8c8f2d53d773f310"}',
+        )
+        sdk = self.get_looker_sdk()  # NOQA
+        # resp = sdk.check_pdt_build(materialization_id=materialization_id)
+
+        self.log.info("Check PDT build response: '%s'.", resp)
+        return resp
+
+    def pdt_build_status(
+        self,
+        materialization_id: str,
+    ) -> Dict:
+        """
+        Gets the PDT materialization job status.
+
+        :param materialization_id: Required. The materialization id to check status for.
+        """
+        resp = self.check_pdt_build(materialization_id=materialization_id)
+
+        status_json = resp['resp_text']
+        status_dict = json.loads(status_json)
+
+        self.log.info(
+            "PDT materialization job id: %s. Status: '%s'.", materialization_id, status_dict['status']
+        )
+
+        return status_dict
+
+    def stop_pdt_build(
+        self,
+        materialization_id: str,
+    ) -> MaterializePDT:
+        """
+        Starts a PDT materialization job cancellation request.
+
+        :param materialization_id: Required. The materialization id to stop.
+        """
+        self.log.info("Stopping PDT materialization. Job id: %s.", materialization_id)
+        self.log.debug("PDT materialization job source: '%s'.", self.source)
+
+        # TODO: Temporary API response stub. Use Looker API once looker sdk 22.2 released.
+        resp = MaterializePDT(
+            materialization_id='366',
+            resp_text='{"success":true,"connection":"incremental_pdts_test",'
+            '"statusment":"KILL 810852",'
+            '"task_slug":"6bad8184f94407134251be8fd18af834"}',
+        )
```

Comment:
If this is necessary then we can use `json.dumps` instead of keeping dict as string

Reply:
same here - I removed this completely as looker sdk 22.2 is finally launched now.